Lesson34/Merve/Lesson33/props-drilling-app/src/modules/user/LogInForm/index.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useContext } from "react";
+import { UserDispatchContext } from "../UserProvider";
+import './styles.css';
+
+export const LogInForm = () => {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const dispatchUser = useContext(UserDispatchContext);
```

Comment:
Since you don't user reducer function but only useState hook in the User Context, it would be more explicit to name this variable `setUser` instead.